From e29f6e549b7c42ebdf181bb079020c736fce1311 Mon Sep 17 00:00:00 2001
From: Joshua Kaldon <joshua@kaldon.com>
Date: Wed, 15 Jan 2025 04:51:03 -0500
Subject: [PATCH] Patch for broken libnfsimapd static and regex plugins.

It appears that the makefile does not add nfsidmap_common.c in the
sources. nfs-utils (1:2.8.2-1.1~0.1) UNRELEASED; urgency=medium .
Non-maintainer upload.  Fix issue with static and regex plugins
missing symbol get_grnam_buflen.

Signed-off-by: Steve Dickson <steved@redhat.com>
---
 support/nfsidmap/Makefile.am | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/support/nfsidmap/Makefile.am
+++ b/support/nfsidmap/Makefile.am
@@ -40,15 +40,15 @@ nsswitch_la_SOURCES = nss.c nfsidmap_com
 nsswitch_la_LDFLAGS = -module -avoid-version
 nsswitch_la_LIBADD = ../../support/nfs/libnfsconf.la
 
-static_la_SOURCES = static.c
+static_la_SOURCES = static.c nfsidmap_common.c
 static_la_LDFLAGS = -module -avoid-version
 static_la_LIBADD = ../../support/nfs/libnfsconf.la
 
-regex_la_SOURCES = regex.c
+regex_la_SOURCES = regex.c nfsidmap_common.c
 regex_la_LDFLAGS = -module -avoid-version
 regex_la_LIBADD = ../../support/nfs/libnfsconf.la
 
-umich_ldap_la_SOURCES = umich_ldap.c
+umich_ldap_la_SOURCES = umich_ldap.c nfsidmap_common.c
 umich_ldap_la_LDFLAGS = -module -avoid-version
 umich_ldap_la_LIBADD = -lldap $(KRB5_GSS_LIB) ../../support/nfs/libnfsconf.la
 
